packages/next/build/utils.ts

```diff
@@ -242,13 +242,15 @@ export async function getCacheIdentifier({
 export async function getPageSizeInKb(
   page: string,
   distPath: string,
-  buildId: string
+  buildId: string,
+  target: string
 ): Promise<number> {
-  const clientBundle = path.join(
-    distPath,
-    `static/${buildId}/pages/`,
-    `${page}.js`
-  )
+  let clientBundle
+  if (target === 'serverless' && page.startsWith('/api')) {
+    clientBundle = path.join(distPath, 'serverless/pages', `${page}.js`)
```

Comment:
We don't need API bundle sizes -- we only care about the client bundles. You can revert these changes.

Reply:
@timneutkens said that we should display it

Reply:
I said display the modules etc, the size is fine to be n/a